README.md

```diff
@@ -132,15 +132,6 @@ cd $GOPATH/src/github.com/kubeedge/kubeedge
 make # or `make edge_core`
 ```
 
-### Integrate with HuaweiCloud [Intelligent EdgeFabric (IEF)](https://www.huaweicloud.com/product/ief.html)
-**Note:** The HuaweiCloud IEF is only available in China now.
-
```

Comment:
Can we have a link in README to the guide to Integrate with HuaweiCloud IEF

Reply:
Yes, we have already added it in the previous [PR](https://github.com/kubeedge/kubeedge#documentation).

Reply:
cool

Reply:
@m1093782566 , I checked the link in the documentation . It's pointing to https://github.com/kubeedge/kubeedge/blob/master/docs/guides/message_topics.md.

Can you please update the link ?

Reply:
Nice catch! Fixed.